Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/Instance.cpp

```diff
@@ -377,7 +397,12 @@ namespace AzToolsFramework
             return entityAliases;
         }
 
-        void Instance::GetNestedEntityIds(const AZStd::function<bool(AZ::EntityId)>& callback)
+        size_t Instance::GetEntityAliasCount() const
+        {
+            return m_entities.size();
```

Comment:
same. Don't you need to account for container entity too? So this should be size +1

Reply:
Same